internal/resource-request-operator/suite_test.go

```diff
@@ -1,115 +0,0 @@
-package resourcerequestoperator
```

Comment:
Is it desired this file is deleted? It seems to me it is pretty standard when dealing with tests

Reply:
I have done it to try to fix the problems I am having with tests and now I create the environment with BeforeEach() and destroy it in AfterEach() but at moment seems to be useless even if until now tests always pass here.